pnp/PnpHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace PnpHelpers
+{
+    public static class PnpHelper
+    {
```

Comment:
Will there be a followup PR for methods and desired properties?

Reply:
yes, this is just to get initial feedback on the approach for the helper class.